### PR TITLE
Allow customisation of 'project/' path & 'package.json' file.

### PR DESCRIPTION
**Pull request by henrahmagix**
@incuna/js Please merge, ta!

### CODE REVIEW

**Comment by grahamgilchrist** [comment]
@henrahmagix Can we have some background as to the justification for this change? What is the user case/requirement?

**Comment by henrahmagix** [comment]
A current project is using multiple projects in one repo, so it needs to tell maketextfiles where to look for textfiles: just `destinationFileName` isn't enough. Other than that, these are hardcoded values that should be optional.

**Comment by grahamgilchrist** [comment]
I see, so you are sharing one `Gruntfile` between all the projects, rather than a separate `Gruntfile` for each?

That makes sense :) It's just useful to have the background/user requirements in these PRs so we can trace the reasoning if we look back :)

**Comment by henrahmagix** [comment]
:+1:  Thanks!

**Comment by henrahmagix** [comment]
I've released as [`v1.1.0`](https://github.com/incuna/grunt-maketextfiles/releases/tag/v1.1.0)

File: tasks/maketextfiles.js [hunk 1]
```diff
@@ -20,17 +20,24 @@ module.exports = function (grunt) {
...
         var packageFileName = 'package.json';
-        var projectBasePath = 'project/';
-        var projectPackage = grunt.file.readJSON(packageFileName);
-        //template file relative to this file
-        var templateFileName = '../templates/textFiles.template.js';
 
         // Merge task-specific and/or target-specific options with these defaults.
         var options = this.options({
+            projectBasePath: 'project/',
             destinationFileName: projectBasePath + 'textFiles.js'
```
Comment (henrahmagix) [inline tasks/maketextfiles.js:28]: @grahamgilchrist Woops! Bug here: `projectBasePath` is undefined at this point. Fixed and I've released as [`v1.1.0`](https://github.com/incuna/grunt-maketextfiles/releases/tag/v1.1.0)

**Pull request merged**